tests/integration/utils2.go

```diff
@@ -418,6 +418,9 @@ func executeTestCase(
 		case SchemaPatch:
 			patchSchema(s, action)
 
+		case SetDefaultSchemaVersion:
```

Comment:
I think we'll want this action included in the change detector setup steps.

Reply:
I don't think we do.  Normal tests will just set the PatchSchema change to default using the bool, tests that use this action are perhaps going to benefit more from having this in the main (post setup) stage, as they are more likely to be testing that SetDefaultSchemaVersion works.